tasks/default.yml

```diff
@@ -75,6 +75,5 @@
     name: "{{ item }}"
     state: present
     executable: "{{ anaconda_conda_bin }}"
-  register: conda_install_result
-  changed_when: not (conda_install_result.stdout | search('All requested packages already installed'))
```

Comment:
is this a bug with the conda ansible module?

Reply:
The conda Ansible module captures the actual output of the conda package manager. There is no stdout which could be captured, the conda module handles changes and failures on its own. 
E.g.
https://github.com/andrewrothstein/ansible-anaconda/blob/master/subrepos/udst.ansible-conda/conda.py#L99

This module behaves just like the core Ansible module yum does as an example.